docs/src/content/docs/reference/helm-chart-config.mdx

```diff
@@ -164,6 +164,77 @@ The configuration for the Helm chart is provided as a YAML file. It has the foll
     </table>
 </div>
 
+## Website Settings
+
+<div className='overflow-x-scroll'>
+    <table className='min-w-[700px]'>
+        <thead>
+            <tr>
+                <th className='w-56'>Field</th>
+                <th>Type</th>
+                <th className='w-32'>Default</th>
+                <th>Description</th>
+            </tr>
+        </thead>
+        <tbody>
+            <tr>
+                <td>`website`</td>
+                <td>Object</td>
+                <td></td>
+                <td>Website specific setting</td>
+            </tr>
+            <tr>
+                <td>`website.websiteConfig`</td>
+                <td>Object</td>
+                <td></td>
+                <td>Settings for the `website_config.json`</td>
```

Comment:
It seems a bit strange to have a field `websiteConfig` in the config under the key `website` as I'd expect everything under `website` in a config file to be "website config". What about calling it something like `features` as this it defines which features are activated and which not?

Reply:
I agree that it seems weird - but I still think it's a good approach:
* I tried to cluster the config values a bit (instead of having everything lop-level)
* `features` would also be a good, but a different slicing of the options. (slicing by features instead of by where the feature lives). If we go for that, I would propose to restructure other parts of the config in that direction as well.
  * we already have `seqSets.enabled`. That's why I decided for `website` (and I can imagine that one gets similar things for `backend` - targeting the backend only - and other feature groups such as `seqSets` and `auth`.
  * Also some features (like `seqSets`) might need additional configuration parameters, which makes the `features` section almost obsolete again.
* I named it `website.websiteConfig`, because there is also already `website.runtimeConfig`, to distinguish it from that. We could also argue that it's irrelevant for users in which config file those values go and all move them to `website`, but for us as developers I think it's helpful to keep it as an indicator.